src/Analyzers/CSharp/Tests/RemoveUnusedMembers/RemoveUnusedMembersTests.cs

```diff
@@ -1590,6 +1642,39 @@ public async Task PropertyIsTargetOfCompoundAssignmentAndValueDropped()
             await VerifyCS.VerifyCodeFixAsync(code, code);
         }
 
+        [WorkItem(43191, "https://github.com/dotnet/roslyn/issues/43191")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedMembers)]
+        public async Task PropertyIsTargetOfCompoundAssignmentAndValueDropped_VerifyAnalizerMessage()
+        {
+            var code = @"class MyClass
+{
+    private int P { get; set; }
+    public void M1(int x) { P += x; }
+}";
+
+            await VerifyCS.VerifyAnalyzerAsync(code, new DiagnosticResult(
+                CSharpRemoveUnusedMembersDiagnosticAnalyzer.s_removeUnreadMembersRule)
+                .WithSpan(3, 17, 3, 18)
+                .WithArguments("MyClass.P"));
+        }
```

Comment:
need sibling test showing this same code, but something additional that then shows we don't emit this.